fixtures/fixtures.js

```diff
@@ -56,11 +56,14 @@ var Fixtures = {
     },
     voucherStatus: {
         "success":
-            {"success":["RVNT12345678"], "fail":[], "invalid":[]},
+            {"success":["RVNT12345678"], "fail":[], "invalid":[], "message": 'Voucher is valid'},
```

Comment:
success, fail and invalid arrays are no longer being returned by the server. Only message, error and warning. Warning is for 2 types of dupe vouchers.